packages/fetchai/skills/thermometer/temper/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 urwen
```

Comment:
adding the `temper` dir is not ideal imo. Can we use https://pypi.org/project/temper-py/ instead? Then its less code